sito/index.php

```diff
@@ -254,7 +251,7 @@
 							</tr>
 							<tr>
 								<td colspan="4" style="weight: 2em;">
-									<div style="float:left">18:30</div>
+									<div style="float:left;margin-left: 15px;">18:00/18:30</div>

```

Comment:
Il dubbio è fare tutto questo tempo prima della chiusura? dopo la gente se ne va, forse conviene fare alle 18:15 così diamo tempo alle sessioni di chiudere.

Reply:
Anche 18:10 volendo

Reply:
@madbob che dici? così approviamo la pr e aggiorniamo il sito

Reply:
Non credo sia cosi' determinante e vitale essere precisi al minuto sulla tabella degli orari...

Idealmente il secondo round di sessioni si chiude alle 18:00 (16:30 + un'ora e mezza), dunque da programma alle 18:00 si svolgono i saluti finali. Poi sappiamo tutti che non sara' vero, e che si sforera', ma non mi porrei questa problema.

Semmai occorre prevedere come operativamente si gestisce la cosa. I saluti finali si svolgono in una unica sessione streaming (idealmente ricicliamo la sessione 1, che sara' usata anche per la mattina unificata), dunque bisogna fare in modo che il pubblico delle sessioni 2 e 3 cambi pagina verso le 18:00. Eviterei meccanismi automatici in JS, appunto perche' non permettono di gestire gli inevitabili sforamenti e le situazioni non prevedibili, la risolverei in modo prosaico mettendo come background delle rispettive stanze questa immagine:

![end](https://user-images.githubusercontent.com/166089/118966244-d1f2ff80-b969-11eb-8793-4b4510fc2795.png)

Reply:
Mi sembra la soluzione migliore, intanto ho tolto il codice JS.
Per la questione sforamento per questo avevo messo i due orari con lo slash per far capire inizio e fine.

Reply:
come vogliamo fare lasciamo come nella PR? @madbob

Reply:
In tutta la tabella ci sono sempre e solo gli orari di inizio, lascerei anche qui solo "18:00".
Poi sforiamo, lo sappiamo tutti, ma da programma i saluti finali iniziano alle 18:00 e quello ci va scritto.